src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -556,6 +562,9 @@
         - name: --max-count
           type: int
           short-summary: Maximum nodes count used for autoscaler, when "--enable-cluster-autoscaler" specified. Please specify the value in the range of [1, 100]
+        - name: --max-surge
+          type: string
+          short-summary: int (5) or percent (33%)  indidcating the extra nodes to use to speed upgrade
         - name: --mode
```

Comment:
why we need it when updating?

Reply:
This is a long lived setting like the autoscaler settings. You may choose to set it on update/create so you don't have to set it on each upgrade.